### PR TITLE
Add pear packaging module to manage PHP PEAR an PECL packages

### PR DESCRIPTION
**Pull request by jle64**
Hi,

I would like to suggest the inclusion of this module  that manages PHP PEAR an PECL packages.

Thanks

### CODE REVIEW

File: packaging/language/pear.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            would_be_changed.append(package)
+    if would_be_changed:
+        if state == "absent":
+            state = "removed"
+        module.exit_json(changed=True, msg="%s package(s) would be %s" % (
+            len(would_be_changed), state))
+    else:
+        module.exit_json(change=False, msg="package(s) already %s" % state)
+
+import os
```
Comment (sivel) [inline packaging/language/pear.py]: You have imported `os` 2 times, here and at the top.

File: packaging/language/pear.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        remove_c += 1
+
+    if remove_c > 0:
+
+        module.exit_json(changed=True, msg="removed %s package(s)" % remove_c)
+
+    module.exit_json(changed=False, msg="package(s) already absent")
+
+
+def install_packages(module, state, packages, package_files):
```
Comment (sivel) [inline packaging/language/pear.py]: `package_files` does not appear to be used in this function

File: packaging/language/pear.py [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+DOCUMENTATION = '''
+---
+module: pear
+short_description: Manage pear/pecl packages
+description:
+    - Manage PHP packages with the pear package manager.
+author:
+    - "'jonathan.lestrelin' <jonathan.lestrelin@gmail.com>"
+notes: []
```
Comment (bcoca) [inline packaging/language/pear.py]: no need for empty notes or requirements

File: packaging/language/pear.py [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: pear
+short_description: Manage pear/pecl packages
+description:
```
Comment (bcoca) [inline packaging/language/pear.py:27]: missing version added (2.0)

File: packaging/language/pear.py [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+author:
+    - "'jonathan.lestrelin' <jonathan.lestrelin@gmail.com>"
+notes: []
+requirements: []
+options:
+    name:
+        description:
+            - Name of the package to install, upgrade, or remove.
+        required: true
+        default: null
```
Comment (bcoca) [inline packaging/language/pear.py]: no need for default on required fields

**Comment by jle64** [comment]
I believe I've fixed everything listed.

**Pull request merged**